src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -338,25 +380,29 @@ public VSCompletionItemData(string displayText, ImageElement icon, ImmutableArra
 
             public ImmutableArray<AsyncCompletionData.CompletionFilter> Filters { get; }
 
+            public int FilterSetData { get; }
```

Comment:
comment on what does the number represent

Reply:
Fixed